frontend/src/metabase/query_builder/containers/ArchiveQuestionModal.jsx

```diff
@@ -34,10 +34,22 @@ class ArchiveQuestionModal extends Component {
       ? t`This model will be removed from any dashboards or pulses using it.`
       : t`This question will be removed from any dashboards or pulses using it.`;
 
+    const widgetCount = question.getDropdownWidgetCountForValues();
+
+    const additionalWarning =
+      widgetCount > 0 && !isModel
+        ? " " +
+          ngettext(
+            msgid`It will also be removed from the filter that uses it to populate values.`,
+            `It will also be removed from the ${widgetCount} filters that use it to populate values.`,
+            widgetCount,
+          )
+        : "";
+
     return (
       <ArchiveModal
         title={title}
-        message={message}
+        message={message + additionalWarning}
```

Comment:
`${message}{additionalWarning}`

Reply:
I could only make this work as

```
message={`${message}${additionalWarning}`}
```
Am I missing a trick?